use_cases/eluc/predictors/predictor.py

```diff
@@ -45,9 +45,9 @@ def save(self, path: str):
         :param path: path to save the model
         """
 
-
+    @classmethod
     @abstractmethod
-    def load(self, path: str):
+    def load(cls, path: str):
```

Comment:
What does the `load` method return? I expect a `Predictor` class on which I can call the abstract `predict` method.
```
def load(cls, path: str) -> Predictor:
```